tests/providers/google/cloud/operators/test_dataproc.py

```diff
@@ -144,6 +144,26 @@ def test_image_version(self):
             )
             assert "custom_image and image_version" in str(ctx.value)
 
+    def test_custom_image_family_1(self):
```

Comment:
Let's give it a more meaningful test name
```suggestion
    def test_custom_image_family_with_image_version(self):
```